### ISSUE TITLE
Play/pause keyboard shortcut

### ISSUE THREAD

**Issue by dschep** [maxence-charriere/lofimusic#20]
It'd be nice if I could tap the space bar to play & pause the station when the app is focused, like most media players support.

**Comment by Firefox289** [comment]
yeah i think also can adjust the volume with right/left arrow keys and switch songs with up/down arrow keys
